drivers/wireless/lpwan/rn2483/rn2483.c

```diff
@@ -58,10 +58,12 @@
 #define CONFIG_LPWAN_RN2483_FREQ 433050000
 #endif /* CONFIG_LPWAN_RN2483_FREQ */
 
-#if (CONFIG_LPWAN_RN2483_FREQ < 433000000 ||                                           \
-     (CONFIG_LPWAN_RN2483_FREQ > 434800000 && CONFIG_LPWAN_RN2483_FREQ < 863000000) || \
+#if (CONFIG_LPWAN_RN2483_FREQ < 433000000 ||                                 \
```

Comment:
Just a sidenote, we should figure out a solution for the formatting as it makes reading what actually changed in these pulls much harder.

Reply:
I agree; we could put a clang-format configuration option in, but the problem is that I have not found a format configuration that matches the NuttX style guide. The best solution is just to disable formatting I think and do it by hand. Kinda sucks so other solutions welcome.